doc/source/tutorial/optimize.rst

```diff
@@ -1347,6 +1350,215 @@ Preconditioning is an art, science, and industry. Here, we were lucky
+
+::
+
+    >>> import numpy as np
+    >>> from scipy.optimize import linprog
+    >>> c = np.array([-29.0, -45.0, 0.0, 0.0]).T
+    >>> A_ub = np.array([[1.0, -1.0, -3.0, 0.0],
+    ...                [-2.0, 3.0, 7.0, -3.0]])
+    >>> b_ub = np.array([[5.0],
+    ...                [-10.0]])
+    >>> A_eq = np.array([[2.0, 8.0, 1.0, 0.0],
+    ...                 [4.0, 4.0, 0.0, 1.0]])
+    >>> b_eq = np.array([[60.0],
+    ...                [60.0]])
+    >>> x0_bounds = (0, None)
+    >>> x1_bounds = (0, 5.0)
+    >>> x2_bounds = (-np.inf, 0.5)  # +/- np.inf can be used instead of None
+    >>> x3_bounds = (-3.0, 0)
+    >>> bounds = [x0_bounds, x1_bounds, x2_bounds, x3_bounds]
+    >>> result = linprog(c, A_ub=A_ub, b_ub=b_ub, A_eq=A_eq, b_eq=b_eq, bounds=bounds)
+    >>> print(result)
+        con: array([42.58855175, 45.54023356])  # may vary
+        fun: -143.26958745795812  # may vary
+    message: 'The algorithm terminated successfully and determined that the problem is infeasible.'
+        nit: 4
+      slack: array([  2.89860769, -12.11677835])  # may vary
+     status: 2
+    success: False
+          x: array([ 2.41503507,  1.62741268, -0.43792331, -1.71002454])  # may vary
+
+The result shows our problem is infeasible. It means that there is no solution vector that satisfies all the
+constraints. What did we do wrong? If we check the code carefully, we can notice that we've set the :math:`x_3` bound
+constraint wrong. We've set the maximum bound for :math:`x_3` to :math:`0`, but it is not needed.
+
+We can solve it with correct bound settings:
+
+::
+
+    >>> x3_bounds = (-3.0, None)
+    >>> bounds = [x0_bounds, x1_bounds, x2_bounds, x3_bounds]
+    >>> result = linprog(c, A_ub=A_ub, b_ub=b_ub, A_eq=A_eq, b_eq=b_eq, bounds=bounds)
+    >>> print(result)
+         con: array([9.78840831e-09, 1.04662945e-08])  # may vary
+         fun: -505.97435889013434  # may vary
+     message: 'Optimization terminated successfully.'  # may vary
+         nit: 4
+       slack: array([ 6.52747190e-10, -2.26730279e-09])  # may vary
+      status: 0
+     success: True
+           x: array([ 9.41025641,  5.17948718, -0.25641026,  1.64102564])  # may vary
+
+The result shows the optimization was succeeded.
+We can check the objective value (``result.fun``) is same as :math:`c^Tx`:
+
+::
+
+    >>> x = np.array([result.x]).T
+    >>> print(c.T @ x)
+    -505.97435889013434  # may vary
+
+We can also check all constraints are almost all satisfied:
+
+::
+
+    >>> print(A_ub @ x)
+    [[  5.]  # may vary
+    [-10.]]  # may vary
+    >>> print(b_ub)
+    [[  5.]  # may vary
+    [-10.]]  # may vary
+    >>> print(A_eq @ x)
+    [[59.99999999]  # may vary
+    [59.99999999]]  # may vary
+    >>> print(b_eq)
+    [[60.]  # may vary
+    [60.]]  # may vary
+    >>> print([0 <= result.x[0], 0 <= result.x[1] <= 6.0, result.x[2] <= 0.5, -3.0 <= result.x[3]])
+    [True, True, True, True]
+
```

Comment:
If we need greater accuracy, typically at the expense of speed, we can solve using the `revised simplex` method:

```
>>> result = linprog(c, A_ub=A_ub, b_ub=b_ub, A_eq=A_eq, b_eq=b_eq, 
                     bounds=bounds, method='revised simplex')
>>> print(result)
     con: array([0.00000000e+00, 7.10542736e-15])  # may vary
     fun: -505.97435897435895    # may vary
 message: 'Optimization terminated successfully.'
     nit: 5
   slack: array([ 1.77635684e-15, -3.55271368e-15])  # may vary
  status: 0
 success: True
       x: array([ 9.41025641,  5.17948718, -0.25641026,  1.64102564])  # may vary

Reply:
I fixed.